src/llvm/lib_llvm/target.cr

```diff
@@ -31,4 +36,5 @@ lib LibLLVM
   fun abi_size_of_type = LLVMABISizeOfType(td : TargetDataRef, ty : TypeRef) : ULongLong
   fun abi_alignment_of_type = LLVMABIAlignmentOfType(td : TargetDataRef, ty : TypeRef) : UInt
   fun offset_of_element = LLVMOffsetOfElement(td : TargetDataRef, struct_ty : TypeRef, element : UInt) : ULongLong
+  fun copy_string_rep_of_target_data = LLVMCopyStringRepOfTargetData(td : TargetDataRef) : Char*
```

Comment:
That function was missing from LibLLVM despite being used by LLVM::TargetData#to_data_layout_string

Reply:
`#to_data_layout_string` needs to use `LLVM.string_and_dispose` as well